src/server/bg_services/bucket_logs_upload.js

```diff
@@ -65,6 +65,24 @@ class BucketLogUploader {
         return true;
     }
 
+    /**
+     * Fetch the log prefix for the source bucket
+     * @param string bucket_name Name os the source bucket.
+     * @return string Configured log prefix for source bucket.
+     */
+    _read_bucket_logging_prefix(bucket_name) {
+        const bucket = system_store.data.buckets.find(bucket => bucket.name.unwrap() === bucket_name);
+        if (!bucket) {
+            dbg.error('Could not find source bucket in system_store: ', bucket_name);
+            return "";
+        }
+        if (!bucket.logging) {
+            dbg.error('Found source bucket but bucket logging was not configured: ', bucket_name);
+            return "";
```

Comment:
this is a legitimate prefix - so we shouldn't use this to mark that we got an error -  We should maybe handle the possible errors in the main function and skip for all of this possible issue.

Reply:
Yes,  will change it.

Reply:
done